examples/PACKAGES/mbx/1024h2o/in.mbx_h2o

```diff
@@ -0,0 +1,58 @@
+processors      * * * map xyz
+
+units           real
+atom_style      full
+boundary        p p p
+
+read_data       initial.data 
+#read_restart    restart.old
+
+pair_style      mbx 9.0
+pair_modify     mix arithmetic
```

Comment:
There does not seem to be a pair style present that supports mixing. Why use the command when it has no effect? It just confuses beginners to think that it would be needed.

```suggestion
```